### ISSUE TITLE
Background image does not show up when server base URL path is specified

### ISSUE THREAD

**Issue by drnguyenn** [andfanilo/streamlit-drawable-canvas#133]
# Summary
The background image does not show up when Streamlit server base URL path is specified (via `config.toml` file, `--server.baseUrlPath` when running, or environment variable `STREAMLIT_SERVER_BASE_URL_PATH`).
Version: 0.9.3

# Current behavior
As aforementioned in Summary.

# Expected behavior
The background image always shows up correctly.

**Comment by awenznero** [comment]
Thanks for your work!
I tried to implement it into my own workspace, sadly the background image still doesnt show up for me when i publish the repo via streamlit-cloud. Mind having a look since there is no lifesign from the lib-owner so far? 
https://fsc-test.streamlit.app/

**Comment by drnguyenn** [comment]
@awenznero Yeah, it seems like he has no time to maintain this project. I'm sorry but I have no experience with streamlit-cloud yet. I've only tested it on my local machine. Maybe routing on the cloud environment works differently?

**Comment by awenznero** [comment]
Thanks for the fast reply!
I guess since the streamlit-cloud was just for testing i will leave it like this.
Later on it should run in a Docker, so lets see.

**Comment by drnguyenn** [comment]
@awenznero I've left in my PR description a Dockerfile that I used to test the changes I've made. You can have a look and try it in your environment to see if it works for you.
